dandi/tests/test_upload.py

```diff
@@ -250,7 +251,11 @@ def test_upload_bids_non_nwb_file(bids_dandiset: SampleDandiset) -> None:
     assert [asset.path for asset in bids_dandiset.dandiset.get_assets()] == ["README"]
 
 
-def test_upload_sync_zarr(mocker, zarr_dandiset):
+@sweep_embargo
+def test_upload_sync_zarr(mocker: MockerFixture, zarr_dandiset: SampleDandiset, embargo: bool) -> None:
```

Comment:
The linter is complaining that this line is too long.  You don't seem to have pre-commit set up in your local clone.